docs/reference/manifest/extensionpoint.md

```diff
@@ -85,7 +85,7 @@ The following examples show how to use the **ExtensionPoint** element with **Pri
 |**Label**|Required. The label of the group. The **resid** attribute can be no more than 32 characters and must be set to the value of the **id** attribute of a **String** element. The **String** element is a child element of the **ShortStrings** element, which is a child element of the **Resources** element.|
 |**Icon**|Required. Specifies the group's icon to be used on small form factor devices, or when too many buttons are displayed. The **resid** attribute can be no more than 32 characters and must be set to the value of the **id** attribute of an **Image** element. The **Image** element is a child element of the **Images** element, which is a child element of the **Resources** element. The **size** attribute gives the size, in pixels, of the image. Three image sizes are required: 16, 32, and 80. Five optional sizes are also supported: 20, 24, 40, 48, and 64.|
```

Comment:
Could we link each element (first column) to the manifest reference page where available? So here Icon would link to the Icon page.